.gitignore

```diff
@@ -1,9 +1,11 @@
 # Project Specific
 /data/
+/models/
 /share/
 /static/
 /db.sqlite3
 /.env
+/.vscode
```

Comment:
@bsekachev , we have a couple of tracked files from .vscode. What is the reason to add it now?

Reply:
@nmanovic 
Oh, forgot about them.